caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -192,6 +196,29 @@ def public_model_info(cls) -> Dict[str, Any]:  # pylint: disable=no-self-argumen
             "sentence_embedding_dimension": cls.model.get_sentence_embedding_dimension(),
         }
 
+    @TokenizationTask.taskmethod()
+    def run_tokenizer(
+        self,
+        text: str,
+    ) -> TokenizationResults:
+        """Run tokenization task against the model
+
+        Args:
+        text: str
+                Text to tokenize
+        Returns:
+            TokenizationResults
+                The token count
+        """
+        result = self.model.tokenizer.encode_plus(text, return_offsets_mapping=True)
```

Comment:
`encode_plus` is deprecated: https://huggingface.co/docs/transformers/v4.41.3/en/internal/tokenization_utils#transformers.PreTrainedTokenizerBase.encode_plus

Can we use `__call__` instead?